sdk/spring/azure-spring-data-cosmos/pom.xml

```diff
@@ -85,10 +85,21 @@
       <artifactId>spring-expression</artifactId>
       <version>6.0.0-M5</version> <!-- {x-version-update;org.springframework:spring-expression;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>io.micrometer</groupId>
+      <artifactId>micrometer-core</artifactId>
+      <version>1.10.0-M6</version> <!-- {x-version-update;io.micrometer:micrometer-core;external_dependency} -->
+    </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
-      <version>4.36.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+      <version>4.35.0</version> <!-- {x-version-update;com.azure:azure-cosmos;dependency} -->
```

Comment:
We should still depend on current version of azure-cosmos like we do right now, so that we can continuously run the integration tests when we develop azure-cosmos module. 
If we don't do this, we won't be able to catch any regressions because of development on azure-cosmos module, which can then only be caught after the azure-cosmos module has been released.